tests/test_init.py

```diff
@@ -2,6 +2,7 @@
 """
 # pylint: disable=C0103
 import warnings
+from black import Line
```

Comment:
I'm not sure if this is used - could we drop the `black.Line` import?

Reply:
Hum I have no idea why it's here. I'll remove it. Thank you for the catch.